docs/source/tutorials/day_of_the_week.qmd

```diff
@@ -197,16 +199,56 @@ hosp_model.run(
     rng_key=jax.random.key(54),
     mcmc_args=dict(progress_bar=False),
 )
+```
+
+```{python}
+# | code-fold: true
+import arviz as az
+import matplotlib.pyplot as plt
+
+
+# Retrieve the posterior samples from the model
+ppc_samples = hosp_model.posterior_predictive(
+    n_datapoints=daily_hosp_admits.size
+)
+
+# Create an InferenceData object from model
+idata = az.from_numpyro(
+    posterior=hosp_model.mcmc,
+    posterior_predictive=ppc_samples,
+)
 
-# Plotting the posterior
-out = hosp_model.plot_posterior(
-    var="latent_hospital_admissions",
-    ylab="Hospital Admissions",
-    obs_signal=daily_hosp_admits.astype(float),
+# Convert hospital admissions from discrete to continuous
+idata.posterior_predictive["negbinom_rv"] = idata.posterior_predictive[
+    "negbinom_rv"
+].astype(float)
+
+# Use a time series plot (plot_ts) from arviz for plotting
+axes = az.plot_ts(
+    idata,
+    y="negbinom_rv",
+    y_hat="negbinom_rv",
+    num_samples=150,
+    y_kwargs={"color": "red", "linewidth": 2.5},
```

Comment:
The color scheme is a little intense.

Reply:
<img width="632" alt="Screenshot 2024-10-02 at 15 47 22" src="https://github.com/user-attachments/assets/3c77bf8c-8aa1-49d2-8c2e-d3b928c10084">

This is what it looks like (not sure if you were looking at the code after running, so attaching image).

Reply:
I also prefer for observed values to be dots rather than lines, but that's optional.

Reply:
> I also prefer for observed values to be dots rather than lines, but that's optional.

I didn't know this, but will change it.

Reply:
I suggest using one of the arviz color schemes https://python.arviz.org/en/stable/examples/styles.html

Reply:
Some plot styles (for variables not in `observed_data`) have styles I've chosen. For these plots, what would you like besides "a seemingly harsh" color scheme?

Reply:
This sounds good.

Reply:
I think you could specify `"color": "skyblue" ` and it would look decent https://matplotlib.org/stable/gallery/color/named_colors.html